notebooks/03-query/05-lookup-essentials.ipynb

```diff
@@ -0,0 +1,1560 @@
+    "\"PWM\": \"large_airport\",\n",
+    "\"RAP\": \"medium_airport\",\n",
+    "\"RDD\": \"medium_airport\",\n",
+    "\"RDM\": \"medium_airport\",\n",
+    "\"RDU\": \"large_airport\",\n",
+    "\"RIC\": \"large_airport\",\n",
+    "\"RNO\": \"large_airport\",\n",
+    "\"ROA\": \"large_airport\",\n",
+    "\"ROC\": \"large_airport\",\n",
+    "\"RST\": \"large_airport\",\n",
+    "\"RSW\": \"large_airport\",\n",
+    "\"SAN\": \"large_airport\",\n",
+    "\"SAT\": \"large_airport\",\n",
+    "\"SAV\": \"large_airport\",\n",
+    "\"SBA\": \"medium_airport\",\n",
+    "\"SBN\": \"large_airport\",\n",
+    "\"SBP\": \"medium_airport\",\n",
+    "\"SCC\": \"medium_airport\",\n",
+    "\"UNV\": \"medium_airport\",\n",
+    "\"SDF\": \"large_airport\",\n",
+    "\"SEA\": \"large_airport\",\n",
+    "\"SFO\": \"large_airport\",\n",
+    "\"SGF\": \"large_airport\",\n",
+    "\"SGU\": \"medium_airport\",\n",
+    "\"SHV\": \"medium_airport\",\n",
+    "\"SIT\": \"medium_airport\",\n",
+    "\"SJC\": \"large_airport\",\n",
+    "\"SJT\": \"medium_airport\",\n",
+    "\"SJU\": \"large_airport\",\n",
+    "\"SLC\": \"large_airport\",\n",
+    "\"SMF\": \"large_airport\",\n",
+    "\"SMX\": \"medium_airport\",\n",
+    "\"SNA\": \"large_airport\",\n",
+    "\"SPI\": \"large_airport\",\n",
+    "\"SPS\": \"large_airport\",\n",
+    "\"SRQ\": \"large_airport\",\n",
+    "\"STL\": \"large_airport\",\n",
+    "\"STT\": \"medium_airport\",\n",
+    "\"STX\": \"medium_airport\",\n",
+    "\"SUN\": \"medium_airport\",\n",
+    "\"SWF\": \"medium_airport\",\n",
+    "\"SYR\": \"large_airport\",\n",
+    "\"TLH\": \"large_airport\",\n",
+    "\"TOL\": \"large_airport\",\n",
+    "\"TPA\": \"large_airport\",\n",
+    "\"TRI\": \"large_airport\",\n",
+    "\"TUL\": \"large_airport\",\n",
+    "\"TUP\": \"medium_airport\",\n",
+    "\"TUS\": \"large_airport\",\n",
+    "\"TVC\": \"medium_airport\",\n",
+    "\"TWF\": \"medium_airport\",\n",
+    "\"TXK\": \"medium_airport\",\n",
+    "\"TYR\": \"medium_airport\",\n",
+    "\"TYS\": \"large_airport\",\n",
+    "\"VLD\": \"medium_airport\",\n",
+    "\"VPS\": \"large_airport\",\n",
+    "\"WRG\": \"medium_airport\",\n",
+    "\"XNA\": \"medium_airport\",\n",
+    "\"YAK\": \"medium_airport\",\n",
+    "\"NYL\": \"medium_airport\"\n",
+    "                    }\n",
+    "            }\n",
+    "        }\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1c66c90f-4086-4586-8466-03bbed5d8bcb",
+   "metadata": {},
+   "source": [
+    "Now run this cell to send the object to the endpoint."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "61105a0b-1f97-4b87-8621-1122fcab188a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "postLookup(lookup_definition)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aacfe43d-99aa-4a58-a6ce-ab7481ec21d9",
+   "metadata": {},
+   "source": [
+    "Now run this to see what the status is."
```

Comment:
Agreed. We need to explain that it is asynchronous and that the default settings for the "cluster" uses 2 minute coordinator load cycle for lookups. I updated it to 10 seconds and a few other properties so that the processes load lookup on startup and the common definition of the lookup tier name for all processes to '__default' which I'm not sure is necessary. MSQ lookups now work because the Peon load the lookups on startup. See other comment with sample MSQ ingest using lookups.